controller/iteration_test.go

```diff
@@ -422,8 +462,16 @@ func createSpaceAndRootAreaAndIterations(t *testing.T, db application.DB) (space
 	var rootIterationObj iteration.Iteration
 	var otherIterationObj iteration.Iteration
 	application.Transactional(db, func(app application.Application) error {
+		owner := &account.Identity{
+			ID:           uuid.NewV4(),
```

Comment:
thanks, done in https://github.com/almighty/almighty-core/pull/1347/commits/dd3159268e68f2e0036df69c31f208e104325781